### DISCUSSION TITLE
2006/09/13/%E6%96%AD%E7%82%B9%E5%92%8C%E6%8E%A2%E9%92%88

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#115] [Announcements]
# 2006/09/13/%E6%96%AD%E7%82%B9%E5%92%8C%E6%8E%A2%E9%92%88

1\. 断点

https://blog.qizhen.xyz/2006/09/13/%E6%96%AD%E7%82%B9%E5%92%8C%E6%8E%A2%E9%92%88

**Comment by ruanqizhen** [comment]
之前博客下的留言：


Qizhen
2006年09月18日 09:45 编辑
转载是没有问题的，不过最好给出原文的连接地址，因为很多文章我写的时候比较匆忙，可能会有些错误，我会不断回头修改这些文章的。

回复
强
2006年09月17日 01:22 编辑
hehe首先我先say 声sorry,在没有你的同意下,我已经把你的关于labVIEW全部放到vihome网站上,:)请谅解

回复
强
2006年09月17日 01:04 编辑
这位大虾,您好,整整花了半天的时间浏览了你的大部分blog,尤其是关于labVIEW的,收益非浅吧,谢谢.我是上海普通一名测试工程师,也是对labVIEW比较感兴趣,从学校就开始用,然后毕业的时候,我自发组织了labVIEW技术论坛,过了几个月,我又推出LabVIEW的主站,目的,就是让更多的会员和朋友有这样一个可以互相交流的平台,因为交流实在太重要,特别是labVIEW这种时间还是不是用户很广语言开发平台,你的blog做得很不错,所以我希望你可以加入虚拟仪器之家大家庭来.帮助更多的需要帮助的人:)(www.vihome.com.cn  http://www.vifbbs.com)虚拟小强

回复
深圳leo
2006年09月16日 01:39 编辑
 
 
好专业的空间
 
看得我糊里糊涂的
 
看来需要学习的东西太多!

回复
┦花づ葬┭╀
2006年09月13日 19:48 编辑
是你加的我吗？来看看。
